src/css/feature-cards.css

```diff
@@ -0,0 +1,48 @@
+.card-deck {
```

Comment:
This component could be `feature-card` (singular) and the deck could be moved to the site specific stylesheets.